microsoft-edge/devtools-guide-chromium/whats-new/2023/01/devtools-109.md

```diff
@@ -0,0 +1,222 @@
+
+![Allocation sampling profiling options](./devtools-109-images/allocation-sampling-profiling-options.png)
+ 
+See also:
+* [Investigate memory allocation by function](../../../memory-problems/index.md#investigate-memory-allocation-by-function) in _Fix memory problems_.
+* PR with more details is pending - todo
+
+
+<!-- ====================================================================== -->
+## Try the new Heap Snapshot visualizer extension for the DevTools
+
+<!-- Subtitle: Visualize the data in your heap snapshot like you've never seen before, as a directed graph or a tree. -->
+
+Add the new **Heap Snapshot Visualizer** extension to Microsoft Edge to get new visualizations of the data in your heap snapshot files.  The extension adds a new tool to DevTools: **Heap Snapshot Visualizer**.  You can load your heap snapshot files to see them represented either as a directed graph or as a tree.  These new visualizations enable you to explore the retainers chain from the garbage-collection (GC) root to an individual node.
+
+![Heap Snapshot Visualizer 1](./devtools-109-images/heap-snapshot-visualizer-1.png)
+
+![Heap Snapshot Visualizer 2](./devtools-109-images/heap-snapshot-visualizer-2.png)
+<!--
+Instructions for screenshot
+1. Get the heap snapshot visualizer extension from: Heap Snapshot visualizer - Microsoft Edge Addons
+2. With the extension installed, open DevTools.
+3. Go to the new Heap Snapshot Visualizer tool.
+4. Choose file and select the attached .heapsnapshot file that's attached.
+5. Select the Load button.
+6. The Heap Snapshot Visualizer should show the heap snapshot as a graph. Refer to the attached screenshot.
+7. Take a screenshot.
+8. Now select the Tree button. The extension should now show the heap snapshot as a tree. Refer to the attached screenshot.
+9. Take a screenshot.
+-->
+
+See also:
+* [Heap Snapshot Visualizer](https://microsoftedge.microsoft.com/addons/detail/heap-snapshot-visualizer/fceldlhognbemkgfacnffkdanocidgce) - Microsoft Edge Add-ons Store.
+
+
+<!-- ====================================================================== -->
+## Improvements to Quick View in Focus Mode
+
+<!-- Subtitle: Quick View now allows you to pick any tool, show several tools at once and persists across sessions. -->
+
+We listened to your feedback and improved the **Quick View** options in Focus Mode.  Instead of choosing from a list of predefined tools, you can now pick any tool inside DevTools and show several tools at the same time by clicking a **+** button, much like in the main toolbar of DevTools.
```

Comment:
```suggestion
We listened to your feedback and improved the **Quick View** options in Focus Mode.  Instead of choosing from a list of predefined tools, you can now pick any tool inside DevTools and show several tools at the same time by clicking **More tools** (`+`), much like in the main toolbar of DevTools.
```

Reply:
added mid-sentence icon.  recently rewrote paragraph vs the Outdated view here